TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift

```diff
@@ -112,20 +112,28 @@ extension FactorFacade {
     
     private var networkProvider: NetworkProvider!
     private var keyStorage: KeyStorage!
+    private var keyChain: Keychain!
     private var url: String!
     private var authentication: Authentication!
     private var clearStorageOnReinstall = true
+    private var accessGroup: String?
+    private var userDefaults: UserDefaults!
```

Comment:
it is possible to have these properties lazy so you don't force them? I assume there is a reason for not having them as optionals